public/index.html

```diff
@@ -68,13 +68,15 @@
     <link
       href="https://fonts.googleapis.com/css2?family=Rubik:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap"
       rel="stylesheet"
-    />
+  />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <title>Sparkle</title>
   </head>
 
   <body>
     <noscript>You need to enable JavaScript to use Sparkle.</noscript>
-    <div id="root"></div>
+    <div id="root">
+      Looks like you haven't got our latest code loaded yet. Please wait while we fetch this for the first time.
+    </div>
```

Comment:
It is OOS, and I had reverted it in subsequent commit https://github.com/sparkletown/sparkle/pull/1713/commits/3233d1267578632235f91d057f27f4f4cb7b90a9

Reply:
It would still definitely make sense to sync with @sparkletown/product with and get some actual proper content to display there though (OOS of this PR, but as a priority)

Reply:
Because of the mechanics how Lighthouse and similar tools measure first content load, just by removing that sentence, the metric jumps from 0.5 to over 3 seconds.

 Elected to keep simple "Loading..." text for "performance" reasons that will mimic the Loading screen (will look like unstyled version I guess).

Reply:
Full text message still OOS, but having any text in there is within scope I guess.